package.json

```diff
@@ -10,10 +10,7 @@
   "author": "Botpress, Inc.",
   "license": "AGPL-3.0",
   "private": true,
-  "nlu": {
-    "version": "0.0.2",
-    "downloadURL": "https://github.com/botpress/nlu/releases/download/v0.0.2"
-  },
+  "nlu": "0.0.3",
```

Comment:
I'd be tempted to keep the notation  `nlu: { version: '0.0.3' }` since we never know when we might add other properties. Maybe or maybe not necessary. Your thoughts ?

Reply:
Yes, and it's more explicit also